deepcell_toolbox/utils.py

```diff
@@ -377,3 +380,232 @@ def cleanup():
         return shutil.rmtree(dirpath)
     with cd(dirpath, cleanup):
         yield dirpath
+
+import scipy.signal
```

Comment:
Imports should all be at the top of the file.

Reply:
Done